### ISSUE TITLE
"Removed in Sphinx 3.0" warnings

### ISSUE THREAD

**Issue by hdiogenes** [kytos/sphinx-theme#16]
```
$ make -C tutorials
Running Sphinx v2.0.1
making output directory... done
loading intersphinx inventory from https://docs.kytos.io/python-openflow/objects.inv...
loading intersphinx inventory from https://docs.kytos.io/kytos/objects.inv...
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [dirhtml]: targets for 15 source files that are out of date
updating environment: 15 added, 0 changed, 0 removed
$VIRTUAL_ENV/lib/python3.8/site-packages/sphinx/util/nodes.py:149: FutureWarning:                                            
   The iterable returned by Node.traverse()
   will become an iterator instead of a list in Docutils > 0.16.
  for classifier in reversed(node.parent.traverse(nodes.classifier)):
reading sources... [100%] references                                                                                                       
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
$VIRTUAL_ENV/src/kytos-sphinx-theme/kytos_sphinx_theme/kytos/layout.html:24: RemovedInSphinx30Warning: To modify script_files in the theme is deprecated. Please insert a <script> tag directly in your theme instead.
  {% include "navbar.html" %}
$VIRTUAL_ENV/src/kytos-sphinx-theme/kytos_sphinx_theme/kytos/layout.html:56: RemovedInSphinx30Warning: To modify script_files in the theme is deprecated. Please insert a <script> tag directly in your theme instead.
  {%- block content %}
writing output... [100%] references                                                                                                        
generating indices... genindex
writing additional pages... search
$VIRTUAL_ENV/src/kytos-sphinx-theme/kytos_sphinx_theme/kytos/search.html:21: RemovedInSphinx30Warning: To modify script_files in the theme is deprecated. Please insert a <script> tag directly in your theme instead.
  {% endblock %}

copying static files... done
copying extra files... done
dumping search index in English (code: en) ... done
dumping object inventory... done
build succeeded.

The HTML pages are in _build/dirhtml.

Build finished. The HTML pages are in _build/dirhtml.
```

**Comment by Niehaus** [comment]
I'm running sphinx-theme v3.2.1 and I did not got those warnings, I got another ones, and they can be solved by just replacing 
.. code-block:: bash by .. code-block:: console in the indicated lines. I really don't know if those warnings are related to sphinx version yet

```Running Sphinx v3.2.1
making output directory... done
loading intersphinx inventory from https://docs.kytos.io/python-openflow/objects.inv...
loading intersphinx inventory from https://docs.kytos.io/kytos/objects.inv...
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [dirhtml]: targets for 15 source files that are out of date
updating environment: [new config] 15 added, 0 changed, 0 removed
reading sources... [100%] references                                                                 
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] references                                                                  
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/create_looping_napp.rst:217: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/create_looping_napp.rst:277: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/create_ping_pong_napps.rst:460: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/create_your_napp.rst:290: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/publishing_your_napp.rst:200: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/publishing_your_napp.rst:222: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/restful_statistics.rst:106: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/switch_l2.rst:325: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/switch_l3_part1.rst:340: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
/home/barbara/Documents/Github-repos/documentation/tutorials/napps/switch_l3_part2.rst:527: WARNING: Could not lex literal_block as "bash". Highlighting skipped.
generating indices...  genindexdone
writing additional pages...  searchdone
copying static files... ... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 10 warnings.

The HTML pages are in _build/dirhtml.

Build finished. The HTML pages are in _build/dirhtml.
```

**Comment by hdiogenes** [comment]
Those warnings should already be fixed by 5bec7a99a17f27ce5e4434c3ab0c4d7c433c8314, please pull the latest code and try again.
